.github/workflows/ci.yaml

```
@@ -64,11 +64,11 @@ jobs:
 
       - name: Auto-correct RuboCop issues
         working-directory: sinatra
-        run: bundle exec rubocop -A || true
+        run: bundle exec rubocop -A 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)